.gitlab-ci.yml

```diff
@@ -1091,11 +1098,21 @@ test_full_integration:
     - mv /tmp/build_revisions.env /tmp/stage-artifacts .
 
     # Post job status
-    - apk --update add curl jq
+    - apk --update add curl jq sysstat docker-compose hdparm
     - ${CI_PROJECT_DIR}/scripts/github_pull_request_status pending "Gitlab ${CI_JOB_NAME} started" "${CI_JOB_URL}" "${CI_JOB_NAME}/${INTEGRATION_REV}"
 
+    # Start dockerd in the background
+    - dockerd &
+    # Wait for dockerd to start
+    - |-
+      MAX_WAIT=10
+      while [ ! -e "/var/run/docker.sock" ] && [ $MAX_WAIT -ge 0 ]; do
+        MAX_WAIT=$(($MAX_WAIT - 1))
+        sleep 1
+      done
```

Comment:
So this, in principle, is equivalent to the old call to ` /usr/local/bin/dockerd-entrypoint.sh &`, right?

Reply:
`dockerd-entrypoint.sh` without argument (by default) generates a tls certificate and launches `dockerd` configured to serve the api using TLS on `DOCKER_HOST`. By calling `dockerd` with no arguments will ensure the docker api is served on the default unix socket `/var/run/docker.sock`.